src/prefect/main.py

```diff
@@ -49,9 +49,10 @@
 )
 
 # Configure telemetry
-import prefect.telemetry.bootstrap
+if not os.environ.get("PREFECT__ENABLE_OSS_TELEMETRY"):
+    import prefect.telemetry.bootstrap
 
-prefect.telemetry.bootstrap.setup_telemetry()
+    prefect.telemetry.bootstrap.setup_telemetry()
```

Comment:
Could the short circuit that you need be moved to `setup_telemetry` or `prefect.telemetry.bootstrap` instead of being in `prefect.main`?

Reply:
hmm i had intentionally guarded the imports as well but let me see if i can move it inside

Reply:
I was thinking that the env var you're checking for will be unset in the large majority of cases, so it'd be better to be confined to that module. What's the reason for not importing that module if this environment variable is set?

Reply:
that makes sense, i was just under the impression that the imports themselves for the existing OTEL setup would interfere with the new local telemetry. but i haven't checked yet so that might not be true - will update

Reply:
@desertaxle because im targeting `create_app` directly
```
PREFECT_API_URL=http://localhost:4200/api \
OTEL_SERVICE_NAME=prefect-server \
OTEL_TRACES_EXPORTER=otlp \
OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317 \
OTEL_EXPORTER_OTLP_PROTOCOL=grpc \
OTEL_LOG_LEVEL=debug \
PYTHONPATH=/Users/nate/github.com/prefecthq/prefect/src \
  opentelemetry-instrument \
  uvicorn \
  --app-dir /Users/nate/github.com/prefecthq/prefect/src \
  --factory prefect.server.api.server:create_app \
  --host 127.0.0.1 \
  --port 4200 \
  --timeout-keep-alive 5
```

I actually don't need this short circuit at all, so I removed it